v3/tutorials/07-add-a-pallet/index.mdx

```diff
@@ -131,30 +136,32 @@ substrate-node-template/
 
 ### Importing a Pallet Crate
 
-The first thing you need to do to add the Nicks pallet is to import the `pallet-nicks` crate in your
-runtime's `Cargo.toml` file. If you want a proper primer into Cargo References, please check out
-[their official documentation](https://doc.rust-lang.org/cargo/reference/index.html).
+The first thing you need to do is to import the [`pallet-nicks`](/rustdocs/latest/pallet_nicks) crate in your
+runtime's `Cargo.toml` file. If you want a proper primer into Cargo syntax, please check out
+[its official documentation](https://doc.rust-lang.org/cargo/reference/index.html).
 
-Open `substrate-node-template/runtime/Cargo.toml` and you will see a list of all the dependencies
+Open `substrate-node-template/runtime/Cargo.toml` and you will see all the dependencies
 your runtime has. For example, it depends on the [Balances pallet](/rustdocs/latest/pallet_balances/index.html):
 
 **`runtime/Cargo.toml`**
 
 ```TOML
-[dependencies]
 #--snip--
-pallet-balances = { default-features = false, version = '3.0.0', git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-YYYY-MM' }
+[dependencies.pallet-balances]
+default-features = false
+git = 'https://github.com/paritytech/substrate.git'
+tag = 'monthly-2021-08'
```

Comment:
Is this the version tested with? We should do an end to end test on the latest `monthly-2021-10` per #192

Reply:
I think this is fine. Because in the current context, we are showing an example of what would be shown in `Cargo.toml`. We are not asking people to use this code in the tutorial.